modules/library/src/test/scala/zio/elasticsearch/ElasticAggregationSpec.scala

```diff
@@ -141,6 +141,19 @@ object ElasticAggregationSpec extends ZIOSpecDefault {
             equalTo(Max(name = "aggregation", field = "intField", missing = Some(20.0)))
           )
         },
+        test("min") {
```

Comment:
Can you add aggregationWithAllParams? For encoding too.

Reply:
Test already includes case classes where we cover all fields.